app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if not visitor.get("vaccine"):
+            raise NotVaccinatedError("All friends should be vaccinated")
+        if visitor["vaccine"].get("expiration_date") < datetime.date.today():

```

Comment:
Be careful with date comparisons. The `datetime.date.today()` function returns the current date with the time set to 00:00:00, which may lead to incorrect behavior if the `expiration_date` also contains time information. Consider using `datetime.datetime.now()` instead, or ensure that `expiration_date` is also a date without time.